_episodes/02-basics.md

```diff
@@ -97,7 +104,7 @@ Anything to the right of the '#' symbol is treated as a comment. To a large exte
 We also make use of the built-in 'print()' function.
 
 ~~~
-print("a = ", a, "and b = " , b)
```

Comment:
I think the space is actually important here. The output should be "a = a" rather than "a =a" which is what you'll get if you take out the space.

Reply:
Retracted!

Reply:
note the use of `,` _vs._ `+`, so space after `=` would result in `a =[2-spaces]<value>`.
I'd remove the space before third comma to